detox/index.d.ts

```diff
@@ -789,6 +789,46 @@ declare global {
              */
             setLocation(lat: number, lon: number): Promise<void>;
 
+            /**
+             * (iOS only) Override simulator’s status bar.
+             * @platform iOS
+             * @param {config} config status bar configuration.
+             * @example
+             * await device.setStatusBar({
+             *   time: "12:34",
+             *   // Set the date or time to a fixed value.
+             *   // If the string is a valid ISO date string it will also set the date on relevant devices.
+             *   dataNetwork: "wifi",
+             *   // If specified must be one of 'wifi', '3g', '4g', 'lte', 'lte-a', or 'lte+'.
```

Comment:
Not accurate, this is the current list: `'hide', 'wifi', '3g', '4g', 'lte', 'lte-a', 'lte+', '5g', '5g+', '5g-uwb', '5g-uc'`.

However, this might change in upcoming iOS versions, so I suggest we only require a string only.

Reply:
We can also refer to simctl's manual for the currently supported list (by running `xcrun simctl help status_bar`)